tests/functional/a/abstract/abstract_method.py

```diff
@@ -44,7 +44,16 @@ class AbstractD(AbstractB, metaclass=abc.ABCMeta):
     """
 
 
-class Concrete(Abstract): # [abstract-method]
+class ConcreteA(AbstractC):
+    """
+    Concrete class.
+
+    Should trigger a warning since this class does
```

Comment:
I don't really understand this docstring. Shouldn't this then raise `abstract-method`?